content/en/references/configuration.md

```diff
@@ -304,10 +304,11 @@ To learn more about these configuration options, see [DNS Server]({{< ref "dns-s
 
 | Variable | Example Values | Description |
 | - | - | - |
-| `DNS_ADDRESS` | 0.0.0.0 (default) | Address the LocalStack should bind the DNS server on (port 53 tcp/udp). Value `0` to disable.
-| `DNS_SERVER` | 8.8.8.8 (default) | Fallback DNS server for non-modified queries.
-| `DNS_RESOLVE_IP` | 127.0.0.1 | IP address the DNS integration should return as A record for modified queries. This will override any automatic detection of the proper response IP.
-| `DNS_LOCAL_NAME_PATTERNS` | | Names which should be resolved to the LocalStack IP, as python-compatible regex.
+| `DISABLE_TRANSPARENT_ENDPOINT_INJECTION` | `0` (default in Pro) \| `1` | Whether to disable DNS resolution of AWS hostnames to the LocalStack container. Pro feature. (see [Transparent Endpoint Injection]({{< ref "user-guide/tools/transparent-endpoint-injection" >}}))
+| `DNS_ADDRESS` | `0.0.0.0` (default) | Address the LocalStack should bind the DNS server on (port 53 tcp/udp). Value `0` to disable.
+| `DNS_SERVER` | `8.8.8.8` (default) | Fallback DNS server for non-modified queries.
```

Comment:
better 👍

Reply:
I also don't like `8.8.8.8` as default because that's not really true. Only if no default upstream DNS is detected.

Reply:
Mm good point. Maybe 

```markdown
| `DNS_SERVER` | system or `8.8.8.8` | Fallback DNS server for queries not handled by LocalStack. Uses your system DNS if possible, or `8.8.8.8` otherwise.
```

?

Reply:
I went for "Default upstream DNS or `8.8.8.8` (default)"